dashboard/modules/job/job_head.py

```diff
@@ -69,6 +71,18 @@ def job_exists(self, job_id: str) -> bool:
         status = self._job_manager.get_job_status(job_id)
         return status is not None
 
+    @routes.get("/api/version")
+    async def get_version(self, req: Request) -> Response:
+        resp = VersionResponse(
+            version=CURRENT_VERSION,
```

Comment:
can we add a comment here suggesting the value here is temporary and hardcoded as 0 for now, but subject to change later on ?